### PR TITLE
feat: executor to manage its own networks

### PR DESCRIPTION
**Pull request by christyjacob4**

### CODE REVIEW

**Review by abnegate — COMMENTED** [review]
Since the networks are being created lazily - the first injection (and injections after cleanup) will have to wait for the creation of all networks that don't exist, slowing down a cold-start. Might be better to do on server start when possible? Did we do any benchmarking?

**Review by Meldiron — APPROVED** [review]

**Review by abnegate — APPROVED** [review]

File: app/http.php [hunk 1]
```diff
@@ -1199,17 +1238,23 @@ function (string $runtimeId, ?string $payload, string $path, string $method, arr
...
+    Console::success("Orphan runtimes and networks removal finished.");
 
     // TODO: Remove all /tmp folders starting with System::hostname() -
 
+    /**
+     * Create and store Docker Bridge networks used for communication between executor and runtimes
+     */
+    Console::info('Creating networks...');
+    $createdNetworks = createNetworks($orchestration, $networks);
+    Http::setResource('networks', $createdNetworks);
```
Comment (abnegate) [inline app/http.php:1256]: Should be a function returning $createdNetworks

Comment (christyjacob4) [reply app/http.php]: fixed thanks 👍

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -29,5 +29,5 @@ jobs:
 
     - name: Run Tests
       run: |
-        docker run --rm -v $PWD:/app --network openruntimes-runtimes -w /app phpswoole/swoole:4.8.12-php8.0-alpine sh -c \
+        docker run --rm -v $PWD:/app --network runtimes-1 -w /app phpswoole/swoole:4.8.12-php8.0-alpine sh -c \
```
Comment (byawitz) [inline .github/workflows/tests.yml:32]: Not critical, Is it worth changing to `phpswoole/swoole:5.1.2-php8.3-alpine` to match the executor PHP version

File: app/http.php [hunk 3]
```diff
@@ -261,26 +261,72 @@ function getStorageDevice(string $root): Device
...
+ * @return array<string>
+ */
+function createNetworks(Orchestration $orchestration, array $networks): array
 {
-    Console::log('Cleaning up containers...');
+    $imageName = Http::getEnv('OPR_EXECUTOR_IMAGE') ;
+    if (empty($imageName)) {
+        throw new \Exception('Executor image name is not set');
+    }
+    $containers = $orchestration->list(['ancestor' => $imageName, 'status' => 'running']);
```
Comment (christyjacob4) [inline app/http.php:276]:
```suggestion
    $containers = $orchestration->list(['ancestor' => $imageName]);
```

**Pull request merged**